src/components/PopUpForm/PopUpForm.tsx

```diff
@@ -202,13 +202,24 @@ export function PopUpForm() {
       tagsCommaSeparatedString.push(vaccineTypeString);
     }
 
-    const dateToSend = startOfDay(parseISO(date));
+    const tzOffset =
+      getTimezoneOffset(Intl.DateTimeFormat().resolvedOptions().timeZone) /
+      1000 /
+      60 /
+      60;
+
+    // This is a temporary hack while the backend figures out how to manage timezones properly
+    // at which point we will localize properly
+    const dateToSend = `${format(
```

Comment:
If someone selects "2020-06-20" for a popup date while in EST, the datetime string sent to the api is now: date: "2021-06-20T04:00:00+00:00"